tests/test262-esnext-excludelist.xml

```diff
@@ -668,229 +667,6 @@
   <!-- END - Uncategorized failing tests -->
 
   <!-- regression caused by: Invalid regexp patterns should not throw syntax error during parsing (#4038) -->
```

Comment:
This start comment can be removed and also the end comment for this block.

Reply:
Thanks!